### ISSUE TITLE
Select option appears selected in IE, but model has a null value

### ISSUE THREAD

**Issue by silvl** [ngx-formly/ngx-formly#1625]
**I'm submitting a ...**  (check one with "x")
```
[x ] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request
```

**Current behavior**
On IE, when a select field type is used, the first item in the list appears to be selected but the model does not have a value selected. 

**Expected behavior**
To not show any selection when value not specified in the model

**Minimal reproduction of the problem with instructions**
This is reproducable using the formly examples provided on the web and running in IE.   
https://ngx-formly.github.io/ngx-formly/examples/bootstrap-formly/select

**What is the motivation / use case for changing the behavior?**

**Please tell us about your environment:**
IE 11

* **Angular version:** Version used with example

* **Browser:**  IE 11

* **Language:** [all | TypeScript X.X | ES6/7 | ES5]

* **Node (for AoT issues):** `node --version` =

**Comment by silvl** [comment]
As a workaround, if you provide a "placeholder", it will display the placeholder when nothing is selected.

**Comment by aitboudad** [comment]
related to https://github.com/angular/angular/issues/10010

**Comment by aitboudad** [comment]
fixed in `5.2.0`
